content/docs/cli.md

```diff
@@ -0,0 +1,132 @@
+---
+title: The TinaCMS CLI
+---
+
+The _TinaCMS CLI_ can be used to set up your project with TinaCMS schema configuration, and run a local version of the TinaCMS API (using your file system's content). For a real-world example of how this is being used checkout the [Tina Cloud Starter](https://github.com/tinacms/tina-cloud-starter).
+
+## Installation
+
+The CLI can be installed as a dev dependency in your project.
+
+Npm:
+
+```bash
+npm install --save-dev @tinacms/cli
+```
+
+Yarn:
+
+```bash
+yarn add --dev @tinacms/cli
+```
+
+## Usage
+
+```sh
+> yarn run tinacms
+
+Usage:  command [options]
+
+Options:
+  -V, --version           output the version number
+  -h, --help              display help for command
+
+Commands:
+  server:start [options]  Start Filesystem Graphql Server
+  schema:compile          Compile schema into static files for the server
+  schema:types            Generate a GraphQL query for your site's schema, (and
+                          optionally Typescript types)
+  help [command]          display help for command
+```
+
+## Getting started
+
+The simplest way to get started is to add a `.tina/schema.ts` file
+
+```
+mkdir .tina && touch .tina/schema.ts
+```
+
+### `defineSchema`
+
+`defineSchema` tells the CMS how to build your content API.
+
+```ts
+// .tina/schema.ts
+import { defineSchema } from '@tinacms/cli'
+
+export default defineSchema({
+  collections: [
+    {
+      label: 'Blog Posts',
+      name: 'post',
+      path: 'content/posts',
+      fields: [
+        {
+          type: 'string'
+          label: 'Title',
+          name: 'title',
+        },
+      ],
+    },
+  ],
+})
+```
+
+## Run the local GraphQL server
+
+Let's add some content so we can test out the GraphQL server
+
+#### Add an author
+
+```sh
+mkdir content && mkdir content/authors && touch content/authors/napolean.md
```

Comment:
```suggestion
mkdir content && mkdir content/authors && touch content/authors/napoleon.md
```

nit: we have this typo in our starter for a while now ([source](https://en.wikipedia.org/wiki/Napoleon_Dynamite))